src/pages/console/data/content-management.mdx

```diff
@@ -125,7 +125,9 @@ This message occurs when there are records in the selected data model that have
 
 ### Multi-tab limitation
 
-Studio's Data Manager requires use of indexedDB in order to function. Because of this, Data Manager may not load as expected if opened in multiple tabs or windows in your browser. For expected performance of Data Manager, ensure it is only open in one tab at a time.
+If your GraphQL API is using DataStore, Studio's Data Manager requires use of indexedDB (a local database in your web browser) in order to function. Because of this, Data Manager may not load as expected if opened in multiple tabs or windows in your browser. 
```

Comment:
The I should probably be capitalized in `IndexedDB`

Reply:
Fixed